### DISCUSSION TITLE
удаление  обхода запрета

### DISCUSSION THREAD

**Discussion by triplemided** [bol-van/zapret#470] [General]
добрый день,столкнулся с такой проблемой,что не знаю как удалить его
скачивал обход запрета на дискорд и там был указано ,что сделано на основе вашего репозитория
+запрет распространился не только на пк ,но и на вай фай
прошу помочь решить проблему

**Comment by numbereleven-a** [comment]
у оригинального запрета есть скрипт uninstall_easy.sh в корне его и запускать
а что там на основе переделано хз, лучше там спросить где качали если не поможет скрипт от оригинала
